### PR TITLE
Move public episodes to this repo

### LINKED ISSUE #181: Issue running playgrounds locally
First of all, I really love the work you guys are doing to help everyone learn FP concepts in Swift. It's been immensely helpful.

In order to better understand what, why and how you are writing software. I'm able to run the server locally using `make bootstrap-oss` and `make run-oss` however however I've run into one issue when trying to build the server in xCode so I can see how you are using playground-driven development for the view layer.

I get the following error when trying to build the server in xCode (On line 40 of `main.swf`).

```
AppEnvironment.push(\.episodes .~ allEpisodes) // Use of unresolved identifier 'allEpisodes'
```

How can I resolve this so I can play with the playgrounds implementation?

Thanks!

### PR DESCRIPTION
**Pull request by mbrandonw**
As we make episodes permanently public we should move them to this repo so that people can see them in local development. I also fixed #181 by making a separate OSS config.

### CODE REVIEW

File: Sources/Server/main.swift [hunk 1]
```diff
@@ -37,7 +37,22 @@ AppEnvironment.push(\.envVars .~ envVars)
 
 // Transcripts
 
-AppEnvironment.push(\.episodes .~ allEpisodes)
+#if OSS
+  private let allEpisodes = allPublicEpisodes
```
Comment (mbrandonw) [inline Sources/Server/main.swift]: this should help with #181

**Comment by mbrandonw** [comment]
> Nice! Simpler than I thought! Do we need to update the README?

I don't think so. The readme says to run `make boostrap-oss` which should do everything.

**Comment by mbrandonw** [comment]
Just figuring out running tests with OSS configuration and then this should be good to go.

File: Sources/PointFree/PublicEpisodes/0001-Functions.swift [hunk 2]
```diff
@@ -0,0 +1,848 @@
+import Foundation
+import PointFree
```
Comment (stephencelis) [inline Sources/PointFree/PublicEpisodes/0001-Functions.swift]: Can remove this import.

Comment (mbrandonw) [reply Sources/PointFree/PublicEpisodes/0001-Functions.swift]: ah yes good catch

**Pull request merged**